src/printer/tptp/tptp_printer.cpp

```diff
@@ -55,11 +55,22 @@ void TptpPrinter::toStream(std::ostream& out, const CommandStatus* s) const
 
 void TptpPrinter::toStream(std::ostream& out, const Model& m) const
 {
-  out << "% SZS output start FiniteModel for " << m.getInputName() << endl;
+  std::stringstream statusName;
```

Comment:
Nit: This doesn't need to be an `std::stringstream`, it can just be an `std::string`. E.g. `std::string statusName(m.isKnownSat() ? "FiniteModel" : "CandidateFiniteModel")`.